### ISSUE TITLE
Plugins issue (1.7.10)

### ISSUE THREAD

**Issue by sc0rich** [Chicken-Bones/NotEnoughItems#403]
I'm having problems with NEI displaying the buttons on the left side of the screen (time change, weather change, etc) when going into the item menu. when I command into creative, the whole GUI setup for NEI disappears completely. In either creative or survival, NEI cheat mode doesn't work. Though Recipe mode works fine so I didn't suspect anything wrong at first. 
When I logged in last time to check if my mod loadout works, I noticed that NEI was saying there are no plugins loaded. I know I have some plugins that don't have their other dependent mods loaded to latch on to, but like even WAILA wasn't being recognized and WAILA was loaded and working. 
I know it's something I'm doing wrong, but I don't know what it is. I included the latest log (but I didn't go in-game this time), but I'll re-log in and try to mess with NEI a little to get a log on that, and post that log in a later comment. Hopefully it will outline a more holistic picture of the problem I'm having. 

TL:DR; cheat mode doesn't work and NEI isn't reading the plugins.
[latest log 6_16_20_1249am.txt](https://github.com/Chicken-Bones/NotEnoughItems/files/4784658/latest.log.6_16_20_1249am.txt)

**Comment by sc0rich** [comment]
some more logs, once I went in and played with NEI some more in game. Hope this helps.
[launcher_log.txt](https://github.com/Chicken-Bones/NotEnoughItems/files/4784980/launcher_log.txt)
[updateLog.txt](https://github.com/Chicken-Bones/NotEnoughItems/files/4784981/updateLog.txt)
[options.txt](https://github.com/Chicken-Bones/NotEnoughItems/files/4784984/options.txt)
[minetweaker.log](https://github.com/Chicken-Bones/NotEnoughItems/files/4784985/minetweaker.log)
[latest.log](https://github.com/Chicken-Bones/NotEnoughItems/files/4784989/latest.log)

**Comment by sc0rich** [comment]
-_- ok, I just opened the config and I gotta ask
what is the cheatmode and lockmode supposed to be set to? because I'm having the same problems as issue 376, took a look in the configs, and found that the cheatmode was on 2 and lockmode was on -1. this is my first time in the configs file, so that means that when you download NEi 1.0.5.120 from curseforge, it's cheatmode is automatically set to 2, and lockmode automatically set to -1...why? why lock people out of functionality be default? is it because I haven't "moved on" to later versions? I plan to, I just want to play a file with witchery and thaumcraft 4 first. Like, when these mods were in their heyday in 2014, I couldn't even play MC (homeless @ the time), so sue me for not being on the bleeding edge of the mod scene.
